.github/workflows/buildandtest.yml

```diff
@@ -0,0 +1,95 @@
+name: BuildAndTest
+
+on:
+  push:
+    branches: [ development ]
+  pull_request:
+    branches: [ development ]
```

Comment:
Does this mean that CI only runs on PRs and subsequent commits in that PR and not on branches without a PR?

Reply:
yes, that's my understanding